api/tests/integration/ref/basic/test_smiles.py.out

```diff
@@ -121,6 +121,166 @@ c1cc2-c3c(-c2cc1)cccc3
  12  7  4  0  0  0  0
 M  END
 
+F[Po](I)(Cl)Br
```

Comment:
why don't we save information about input stereo?

Reply:
Created separate ticket: https://github.com/epam/Indigo/issues/1129